packages/firebase_inappmessaging/lib/firebase_inappmessaging.dart

```diff
@@ -0,0 +1,25 @@
+import 'dart:async';
+
+import 'package:flutter/services.dart';
+
+class FirebaseInAppMessaging {
+  static const MethodChannel _channel =
+      const MethodChannel('plugins.flutter.io/firebase_inappmessaging');
+
+  static Future<void> triggerEvent(String eventName) async {
+    await _channel.invokeMethod('triggerEvent', {'eventName': eventName});
+  }
+
+  static Future<void> configure({bool suppress, bool dataCollectionEnabled}) async {
```

Comment:
I think we should remove this method and use the setters below.

Reply:
Done!